Update cli.md (#661)
preactjs/preact-www: content/en/cli.md

- 100/100 Lighthouse score, right out of the box ([proof](https://googlechrome.github.io/lighthouse/viewer/?gist=142af6838482417af741d966e7804346)).
- Fully automatic code splitting for routes.

> - Using prefix "async!"  like `import Foo from 'async!./foo';`, Foo Component is lazy-loaded.

Reviewer: ```suggestion
- Lazy-load components by prefixing their import: `import Foo from 'async!./foo'`
```

> - Generating two script tags for browser supporting to [module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and browser not supporting. ([More information](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/))

Reviewer: ```suggestion
- Outputs modern JavaScript with legacy fallback using the [module/nomodule pattern](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/)
```
